test/rpc-server/execute/branching/state.json

```diff
@@ -65,7 +65,7 @@
             "args": [
               {
                 "tag": "EVar",
-                "name": "X",
```

Comment:
Modified this test to have a `VarX` both in the rule and the configuration. With this change, we get the correct `rule-substitution` term in the response:

```json
{
  "tag": "Equals",
  "argSort": {
    "tag": "SortApp",
    "name": "SortInt",
    "args": []
  },
  "sort": {
    "tag": "SortApp",
    "name": "SortGeneratedTopCell",
    "args": []
  },
  "first": {
    "tag": "EVar",
    "name": "RuleVarX",
    "sort": {
      "tag": "SortApp",
      "name": "SortInt",
      "args": []
    }
  },
  "second": {
    "tag": "EVar",
    "name": "VarX",
    "sort": {
      "tag": "SortApp",
      "name": "SortInt",
      "args": []
    }
  }
}
```